app/models/root-store/root-store.ts

```diff
@@ -91,11 +93,12 @@ export const RootStoreModel = types
     }),
   }))
   .actions(self => ({
-    handleUnauthenticatedError() {
+    handleUnauthenticatedError(type: string, error: any) {
+      logError(error)
```

Comment:
Would logging as analyitcs event fits better? This event might be expected depending on cookie expiry, and we cannot handle this "error" in normal circumstances